pkg/osvscanner/osvscanner.go

```diff
@@ -671,7 +671,15 @@ func filterPackageVulns(r reporter.Reporter, pkgVulns models.PackageVulns, confi
 	var newVulns []models.Vulnerability
 	if len(newGroups) > 0 { // If there are no groups left then there would be no vulnerabilities.
 		for _, vuln := range pkgVulns.Vulnerabilities {
-			if _, filtered := ignoredVulns[vuln.ID]; !filtered {
+			unimportant := false
+			for _, affected := range vuln.Affected {
+				if affected.EcosystemSpecific["urgency"] == "unimportant" {
```

Comment:
Can we add a check to only do this for Debian? And probably break this out into a separate function.

e.g.

```golang
func isUnimportant(ecosystemSpecific map[string]interface{}) bool {
}
```